### PR TITLE
Update running_ssh_service.md

### PR DESCRIPTION
**Pull request by gsantandrea**
### Proposed changes
Added a note in  the tutorial for using an older version of Ubuntu (14.04 )

### CODE REVIEW

File: engine/examples/running_ssh_service.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ The following `Dockerfile` sets up an SSHd service in a container that you
 can use to connect to and inspect other container's volumes, or to get
 quick access to a test container.
 
+> *** NOTE: *** For Ubunt 14.04, use  `without-password` instead of `prohibit-password`.
```
Comment (kolyshkin) [inline engine/examples/running_ssh_service.md]: s/Ubunt/Ubuntu/

Comment (ahh-docker) [reply engine/examples/running_ssh_service.md]: Fixed

**Comment by ahh-docker** [comment]
@kolyshkin Lemme know if this is ok with you, and I'll merge it. Thanks!

**Comment by bermudezmt** [comment]
Closing in favor of https://github.com/docker/docker.github.io/pull/8449

**Pull request closed without merging**